.github/workflows/api.yaml

```diff
@@ -86,7 +80,6 @@ jobs:
         uses: irongut/CodeCoverageSummary@v1.3.0
         with:
           header: API Coverage
-          filename: services/api/coverage.xml
```

Comment:
It looks like `filename` is required by the [CodeCoverageSummary action](https://github.com/irongut/CodeCoverageSummary#inputs). So we'll probably need something like this for the action to run:

```yaml
        with:
          header: API Coverage
          filename: coverage.xml
```

Reply:
Oh, oops. I deleted that by mistake. 🤦🏻